pkg/apis/pxc/v1/pxc_types.go

```diff
@@ -169,7 +169,9 @@ type PodSpec struct {
 	TerminationGracePeriodSeconds *int64                                  `json:"gracePeriod,omitempty"`
 	ForceUnsafeBootstrap          bool                                    `json:"forceUnsafeBootstrap,omitempty"`
 	ServiceType                   corev1.ServiceType                      `json:"serviceType,omitempty"`
+	ReplicasServiceType           corev1.ServiceType                      `json:"replicasServiceType,omitempty"`
 	ExternalTrafficPolicy         corev1.ServiceExternalTrafficPolicyType `json:"externalTrafficPolicy,omitempty"`
+	ReplicasExternalTrafficPolicy corev1.ServiceExternalTrafficPolicyType `json:"replicasExternalTrafficPolicy,omitempty"`
```

Comment:
why it is needed to have different ExternalTrafficPolicy for both services?

Reply:
This option depends on ServiceType e.g. you can set  externalTrafficPolicy=Local only for  NodePort and LoadBalancer services. In our case we have two different options (ServiceType and ReplicasServiceType  ) for two services and that is why we need to have ReplicasExternalTrafficPolicy to avoid conflicts.